cluster/README.md

```diff
@@ -0,0 +1,147 @@
+# Cluster
+
+## Getting Started
+
+This module provides an executable for starting a demo cluster of nodes.
+It is designed to remove all the overhead of setting up a configuration
+and an environment and to _just work_, out-of-the-box. Minor configuration
+adjustments are however possible via environment variables. 
+
+```
+$> stack exec -- cardano-sl-cluster-demo
+Cluster is starting (4 core(s), 1 relay(s), 1 edge(s))...
+...core0 has no health-check API.
+......system start:  1539179287
+......address:       127.0.0.1:3000
+......locked assets: -
+...core1 has no health-check API.
+......system start:  1539179288
+......address:       127.0.0.1:3001
+......locked assets: -
+...core2 has no health-check API.
+......system start:  1539179289
+......address:       127.0.0.1:3002
+......locked assets: -
+...core3 has no health-check API.
+......system start:  1539179291
+......address:       127.0.0.1:3003
+......locked assets: -
+...relay has no health-check API.
+......system start:  1539179292
+......address:       127.0.0.1:3100
+...wallet OK!
+......system start:  1539179292
```

Comment:
@disassembler tells me the inconsistent start times have been fixed. Let's update the example text here too.